### PR TITLE
Feature/calendar carousel

### PR DESCRIPTION
**Pull request by bml2238**
image
this also moves the milestones to be below the event carousel to match the mockups

### CODE REVIEW

File: backend/src/main/java/com/senior/project/backend/Activity/EventHandler.java
Comment (jts7382) [inline backend/src/main/java/com/senior/project/backend/Activity/EventHandler.java:24]: again we should get the student id from the auth token. We can handle that later

File: frontend/crd/src/app/dashboard/events/events.component.ts
Comment (jts7382) [inline frontend/crd/src/app/dashboard/events/events.component.ts:31]: please remove the logs

**Review by jts7382 — CHANGES_REQUESTED** [review]

File: frontend/crd/src/app/dashboard/events/events.component.html
Comment (jts7382) [inline frontend/crd/src/app/dashboard/events/events.component.html:13]: You should do this with the Oswego logo component. just add an input and set this styling conditionally in the less

**Review by BasketOfEggs — APPROVED** [review]

**Comment by BasketOfEggs** [comment]
Approved for when Justin's suggestions are implemented

File: frontend/crd/src/app/dashboard/events/events.component.html
Comment (bml2238) [reply frontend/crd/src/app/dashboard/events/events.component.html:13]: set the background image as the oswego logo component?

Comment (bml2238) [reply frontend/crd/src/app/dashboard/events/events.component.html:14]: leaving this as a placeholder

**Review by rcb2957 — APPROVED** [review]
Once the merges are fixed.  I approve the pull request.

**Review by jts7382 — APPROVED** [review]

**Pull request merged**
